cypress/e2e/models/migration/migration-waves/migration-wave.ts

```diff
@@ -167,7 +168,7 @@ export class MigrationWave {
         cy.contains(manageApplications).click();
         cy.get(itemsSelectInsideDialog).click();
         cy.contains(button, selectNone).click();
-        cy.get(MigrationWaveView.applicationsSubmitButton).click();
```

Comment:
Can you check if `MigrationWaveView.applicationsSubmitButton` is used in any other places and remove it from the enum if not?

Reply:
Done